integrations/tensorflow/build_tools/bazel/setup_local_llvm.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import os
+import shutil
+
+
+def symlink_dir_children(source_dir, dest_dir):
+  print(f"Symlink children from {source_dir} -> {dest_dir}")
+  os.makedirs(dest_dir, exist_ok=True)
+  for child in os.listdir(source_dir):
+    dest_path = os.path.join(dest_dir, child)
+    if os.path.exists(dest_path):
+      os.unlink(dest_path)
+    os.symlink(os.path.join(source_dir, child), dest_path)
+
+
+this_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I thought best practice in Python was to check 

```python
if __name__ == "__main__":
```

rather than put commands at the top level?

Reply:
For one off things outside of packages, it isn't important.